privacy-protections-popup/privacy-protections-popup-api/src/main/java/com/duckduckgo/privacyprotectionspopup/api/PrivacyProtectionsPopup.kt

```diff
@@ -19,6 +19,7 @@ package com.duckduckgo.privacyprotectionspopup.api
 import kotlinx.coroutines.flow.Flow
 
 interface PrivacyProtectionsPopup {
-    fun setViewState(viewState: PrivacyProtectionsPopupViewState)
     val events: Flow<PrivacyProtectionsPopupUiEvent>
+    fun setViewState(viewState: PrivacyProtectionsPopupViewState)
```

Comment:
The api is not documented. For all `api` modules we have to provide KDoc comments for classes / interfaces / functions. Check other api modules for reference.